views/index.pug

```diff
@@ -0,0 +1,80 @@
+html
+  include partials/head
+  
+  body 
+    h1 Best Music Albums
+
+    
+    h2 GET/albums
+    h3 Method is GET
+    h4 Description
+    p  Returns a list of all the music albums in the catalogue.
+
+    
+    h2 GET/albums/{id}
+    h3 Method is GET
+    h4 Description
+    p  Returns the album associated with the numeric ID provided in the route.
+
+
+    
+    h2 POST/albums
+    h3 Method is POST
+    h4 Body {  Title: 'Take Care', Artist: [ 'Drake'], ReleaseDate: '11/15/2011' Genres: ['Hip-hop']}
+    h5 Description
+    p  Creates a new album from the data provided in the body.
+
+    
+    h2 DELETE/albums/{id}
+    h3 Method is DELETE
+    h4 Description
+    p  Deletes the album associated with the numeric ID provided in the route.
+
+    
+    h2 GET/artists
+    h3 Method is GET
+    h4 Description
+    p  Returns all albums by an artist
+
+
+    h2 POST/artists
+    h3 Method is POST
+    h4 Body {   Artist: [ 'Madonna'], Genres: ['Pop']}
+    h5 Description
+    p  Creates a new artist from the data provided in the body.
+
+
+    
+    h2 DELETE/artists/{id}
+    h3 Method is DELETE
+    h4 Description
+    p  Deletes the artist associated with the numeric ID provided in the route.
+
+    h2 GET/genres
+    h3 Method is GET
+    h4 Description
+    p  Returns all albums by particular genre
```

Comment:
Currently the only way to add a new genre would be to add a new album or artist. Is that he way you want it. Also be careful of not having a put or patch for genres. If I enter an artist with 'Pop' and another with 'pop' are those now two different genres?